src/icons/general/IconSortChevron.js

```diff
@@ -12,7 +12,7 @@ type Props = {
     width?: number,
 };
 
-const IconSortChevron = ({ className = '', color = '#aeaeae', height = 11, title, width = 11 }: Props) => (
+const IconSortChevron = ({ className = '', color = '#aeaeae', height = 12, title, width = 12 }: Props) => (
```

Comment:
To avoid breaking existing dependencies, you may want to override height / width instead of changing it here.